libs/math/tests/math/all_tests/distance/minkowski.cpp

```diff
@@ -0,0 +1,61 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "core/random/lcg.hpp"
+#include "math/distance/minkowski.hpp"
+#include "math/tensor.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace fetch::math::distance;
```

Comment:
Small point - try to remember to wrap test cpp files' contents in an anonymous namespace. This helps to keep things tidy for the linker, and prevents us from shooting ourselves in the foot accidentally. See `lib/vm_modules/tests/unit/vm_modules_tests.cpp` for an example.